src/omotes_sdk/internal/common/broker_interface.py

```diff
@@ -311,10 +311,17 @@ def run(self) -> None:
         self._loop = asyncio.new_event_loop()
         asyncio.set_event_loop(self._loop)
 
+        setup_task = None
         try:
-            self._loop.create_task(self._setup_broker_interface())
+            setup_task = self._loop.create_task(self._setup_broker_interface())
```

Comment:
In limited occassions it may happen that the setup task is removed automatically by Python as no reference to the task is kept.